tests/test_linestring.py

```diff
@@ -111,6 +112,8 @@ def test_numpy_object_array():
     assert ar[0] == geom
 
 
+# TODO(shapely-2.0)
+@pytest.mark.xfail(strict=False, reason="Not yet implemented for Shapely 2.0")
```

Comment:
Maybe use `shapely20_todo` here?

Reply:
Yeah, I initially did, but that one is currently a "strict" xfail, and this test also seemed to pass in some builds (it might depend on the GEOS version, in any case something to verify and clean-up in a follow-up). 
I could also make the general marker less strict, but the strict version is also nice because then you get notified if something starts passing.